.github/workflows/continuous-integration-osx.yml

```diff
@@ -12,19 +12,39 @@ permissions: read-all
 jobs:
   osxci:
     name: osx-ci
-    runs-on: [macos-latest]
 
+    continue-on-error: true
```

Comment:
Why is it part of this PR?
Do we actually want that?

Reply:
We also do that in `continous-integration-linux` file and I found it useful while setting up the new build. We also detect some more failures in different configuration all at once. Of course, it's somewhat of a drive-by change.

Reply:
We also do that in `continous-integration-linux` file and I found it useful while setting up the new build. We also detect some more failures in different configuration all at once. Of course, it's somewhat of a drive-by change.